### PR TITLE
removing question marks in a bid to prevent downloading each font twice

### PR DESCRIPTION
**Pull request by bjfletcher**
... on first load (by o-fonts CSS in our critical CSS, and again by Service Worker)

### CODE REVIEW

**Comment by wheresrhys** [comment]
Ah good spot. *but* it may actually be the case that wherever uses the url _without_ `?` should have them added. Maybe there to address some odd bug.

@quarterto do you know anything about these `?`?

**Comment by wheresrhys** [comment]
Might be a cack-handed nod to this http://blog.fontspring.com/2011/02/the-new-bulletproof-font-face-syntax/

**Comment by quarterto** [comment]
this was for appcache and `?#iefix` in `o-fonts`, which isn't there any more as of Financial-Times/o-fonts#76. go ahead and remove it

File: config/precache.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const appcache = {
 
 const sw = {
 	fonts: appcache.fonts.concat([
-		'/__origami/service/build/v2/files/o-fonts-assets@1.3.0/MetricWeb-Bold.woff?',
+		'/__origami/service/build/v2/files/o-fonts-assets@1.3.0/MetricWeb-Bold.woff',
```
Comment (wheresrhys) [inline config/precache.js]: update to 1.3.2

Comment (bjfletcher) [reply config/precache.js]: @wheresrhys as I mentioned in the other PR, I think this is meant to be FinancierDisplayWeb not MetricWeb

**Pull request merged**